packages/web/README.md

```diff
@@ -22,17 +27,20 @@ Wrappers for interacting with the Concordium node, for the web environment.
     - [GetModuleSource.html](#getmodulesourcehtml)
     - [SendCIS2Transfer.html](#sendcis2transferhtml)
     - [SendCIS2UpdateOperator.html](#sendcis2updateoperatorhtml)
-- [Build](#build)
-    - [Building for a release](#building-for-a-release)
-    - [Publishing a release](#publishing-a-release)
```

Comment:
Where did the build instructions go?

Reply:
The build/release instructions are in the root README.